pkg/cvo/status.go

```diff
@@ -296,14 +297,17 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 			})
 		} else {
 			var message string
+			percentComplete := float32(status.Done) / float32(status.Total)
 			switch {
 			case len(validationErrs) > 0:
 				message = fmt.Sprintf("Reconciling %s: the cluster version is invalid", version)
-			case status.Fraction > 0 && skipFailure:
+			case percentComplete > 0 && skipFailure:
 				reason = progressReason
-				message = fmt.Sprintf("Working towards %s: %.0f%% complete, %s", version, status.Fraction*100, progressShortMessage)
-			case status.Fraction > 0:
-				message = fmt.Sprintf("Working towards %s: %.0f%% complete", version, status.Fraction*100)
+				message = fmt.Sprintf("Working towards %s: %d of %d done (%.0f%% complete), %s", version,
+					status.Done, status.Total, math.Trunc(float64(percentComplete*100)), progressShortMessage)
+			case percentComplete > 0:
+				message = fmt.Sprintf("Working towards %s: %d of %d done (%.0f%% complete)", version,
+					status.Done, status.Total, math.Trunc(float64(percentComplete*100)))
```

Comment:
The `math.Trunc` bits seem new.  Can you explain what they're for in the commit message?  I would have expected the `%.0f` to already be removing any decimal portion.  Explaining here in the PR would be ok too, if you feel like it's not worth rounds of retesting.

Reply:
Updated commit message. `%.0f` rounds.